src/clippy-js-proxy.c

```diff
@@ -372,6 +377,14 @@ js_value_to_value (JSCValue *src, GValue *dest)
     g_warning ("%s unexpected GValue type", __func__);
 }
 
+JSCValue *
+js_value_object_get_property (JSCValue *value, const gchar *property)
+{
+  g_autofree gchar *jsprop = g_strdup (property);
+  str_replace_char (jsprop, '-', '_');
+  return jsc_value_object_get_property (value, jsprop);
```

Comment:
This will always create a copied string even if there's no dashes in it. I wonder if this is worth changing the code, but just wanted to mention it.